db/schema.rb

```diff
@@ -320,6 +320,15 @@
     t.index ["academic_cycle_id"], name: "index_funding_methods_on_academic_cycle_id"
   end
 
+  create_table "hesa_collection_requests", force: :cascade do |t|
+    t.string "collection_reference"
+    t.datetime "requested_at"
+    t.datetime "updates_since"
+    t.string "response_body"
```

Comment:
I think `t.string "response_body"`  might need to be `t.text "response_body"` instead due to the size of the XML.